package.json

```diff
@@ -195,7 +195,7 @@
     "eslint-plugin-prettier": "^3.3.1",
     "firebase": "^7.24.0",
     "firebase-admin": "^9.4.2",
-    "firebase-functions": "^3.15.0",
+    "firebase-functions": "^3.16.0",
```

Comment:
Wondering if it's okay to bump this? Is there anything else that I need to do?

Reply:
Not really, but for this change, don't we need to update https://github.com/firebase/firebase-tools/blob/master/scripts/triggers-end-to-end-tests/functions/package.json instead?

Reply:
Thought it would be better to update the root package since 3.16 includes all storage enhancements.

I've reverted this bump and bumped the test package instead.